.github/workflows/continuous-integration.yml

```diff
@@ -1,6 +1,6 @@
 name: opensim-application
 
-on: [push]
+on: [push, pull_request]
 
```

Comment:
Was this intended for long term or just during testing? We generally try to limit usage of github community resources if possible.

Reply:
This was necessary to make sure the Linux job built properly, but also to align with your other CI which also runs for PRs. I can revert back to only building for push if you wish?

Reply:
ok, let's revert to [push] for now to reduce resource use until we switch to Actions. Thank you.